src/components/organisms/Accordion/Accordion.md

```diff
@@ -11,26 +11,28 @@ const AccordionExample = () => {
   const items = [
     {
       id: 'one',
-      header: 'header one',
+      header: 'header one - default (lead)',
       section: 'section one',
     },
     {
       id: 'two',
-      header: 'header two',
+      header: 'header two - lead',
       section: 'section two',
+      size: 'lead',
     },
     {
       id: 'three',
-      header: 'header three',
+      header: 'header three - regular',
       section: 'section three',
+      size: 'regular',
     },
   ];
 
   return (
     <div aria-label="accordion example">
-      {items.map(({ id, header, section }, index) => (
+      {items.map(({ id, header, section, size }, index) => (
         <div key={id}>
-          <Accordion.Header isOpen={isActiveSection(index)} {...getHeaderProps(id, index)}>
+          <Accordion.Header isOpen={isActiveSection(index)} {...getHeaderProps(id, index)} textSize={size}>
             {header}
```

Comment:
I think on of the advantages of using render props is that you can pass whatever component in the render, so you can customise  like this:
```<Lead>{header}</Leader>```
and then you don't need custom sizes that can lead to more changes in the future. /cc @thegrinder

Reply:
I guess if Accordion Header has a specific set of sizes, it makes sense to add them as a prop. We just have to make sure we limit the amount of variations

Reply:
I agree with @thegrinder here. We may not want too many different variations on the accordion so limiting it by adding the prop seems like a good way for me. It will give us a default way that you can overwrite like @poteirard says here.